src/js/components/Button/StyledButton.js

```diff
@@ -206,8 +206,11 @@ const StyledButton = styled.button`
     props.theme.button &&
     props.theme.button.disabled &&
     disabledButtonStyle(props)}
-  ${props =>
-    props.focus && (!props.plain || props.focusIndicator) && focusStyle()}
+  
+  &:focus {
+    ${props => (!props.plain || props.focusIndicator) && focusStyle()}
+  }
```

Comment:
Interesting, I thought the problem isn't reproducing on the regular Button.

Reply:
It is occurring on both, just confirmed locally